chapter_appendix/aws.md

```diff
@@ -74,8 +74,8 @@ Here, "/path/to/key.pem" is the path of the locally-stored key used to access th
 It is a good idea to update the instance with the latest drivers.
 
 ```
-sudo apt-get update
-sudo apt-get dist-upgrade
+sudo apt update
+sudo apt dist-upgrade
```

Comment:
let's keep apt-get. for instance, "apt dist-upgrade" should be "apt full-upgrade"

Reply:
Missed that one, true:) The two occurrences of `dist-upgrade` in `aws.md` should be the only mistakes tho.
There was already a mix of `apt` and `apt-get`, I just wanted to keep it consistent